### PR TITLE
Frontend refactor: blade

### PR DESCRIPTION
**Pull request by kristjanjansen**

### CODE REVIEW

File: resources/views/component/offer.blade.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{{--
```
Comment (kristjanjansen) [inline resources/views/component/offer.blade.php]: Internally we call "offer" as "flight". Lets keep it consistent. In user facing documentation lets use "flight offer"

File: resources/views/component/travelletter.blade.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<div class="c-travel-letter">
```
Comment (kristjanjansen) [inline resources/views/component/travelletter.blade.php]: What is travel-letter? Blog? Lets keep it consistent and rename blog to something better in one go.

File: config/menu.php [hunk 3]
```diff
@@ -1,92 +1,125 @@
...
-            ],
-            'twitter' => [
-                'route' => 'https://twitter.com/trip_ee',
-                'external' => true,
-            ],
+    'footer-social' => [
+        'facebook' => [
+            'route' => 'https://facebook.com/tripeeee',
+            'external' => true,
+            'modifier' => 'm-facebook',
```
Comment (kristjanjansen) [inline config/menu.php]: Interesting use case, I'd recommend to use 'icon' property instead but let's discuss. Whaterver the result, follow up on #298

File: resources/views/component/destinationinfo.blade.php [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        'text' => 'Malta on tihedalt asustatud saareriik Vahemeres, mis koosneb 3 asustatud ja neljast asustamata saartest',
+        'wiki_route' => '#',
+        'definitions' => [
+            [
+                'term' => 'Rahvaarv',
+                'definition' => '417 600 in'
+            ],
+            [
+                'term' => 'Pindala',
+                'definition' => '316 km&sup2;'
```
Comment (kristjanjansen) [inline resources/views/component/destinationinfo.blade.php]: ² ? :)

File: resources/views/component/footernav.blade.php [hunk 5]
```diff
@@ -0,0 +1,36 @@
+{{--
```
Comment (kristjanjansen) [inline resources/views/component/footernav.blade.php]: ```footer_nav.blade.php``` or ```footer/nav.blade.php```

File: resources/views/component/footersocial.blade.php [hunk 6]
```diff
@@ -0,0 +1,37 @@
+{{--
```
Comment (kristjanjansen) [inline resources/views/component/footersocial.blade.php]: footer_social.blade.php (preferred) or footer/social.blade.php

File: resources/views/component/map.blade.php [hunk 7]
```diff
@@ -0,0 +1,4 @@
+<div class="c-map {{ $modifiers or '' }}">
+    <div class="c-map__location" style="top: {{ $map_top }}; left: {{ $map_left }};"></div>
+    <?php // echo file_get_contents('images/dummy/map.svg'); ?>
```
Comment (kristjanjansen) [inline resources/views/component/map.blade.php]: #333 will add map.svg, then reference that using  tag?

File: config/menu.php (hunk 3 above)
Comment (apani) [inline config/menu.php]: I was not sure how we will do this, so 'icon' sounds fine

File: resources/views/component/news.blade.php [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+<div class="c-news {{ $modifiers or '' }}">
+    <a href="{{ $route }}" class="c-news__image-wrap">
+        <img src="{{ $image }}" alt="" class="c-news__image">
+    </a>
+    <h3 class="c-news__title">
+        <a href="{{ $route }}" class="c-news__title-link">{{ $title }}</a>
+    </h3>
+    <div class="c-news__meta">
+        <p class="c-news__meta-date">{{ $date }}</p>
```
Comment (kristjanjansen) [inline resources/views/component/news.blade.php]: All dates display need to render through```component.date.*```

File: resources/views/component/footernav.blade.php (hunk 5 above)
Comment (apani) [inline resources/views/component/footernav.blade.php]: lets go with footer/nav.blade.php

File: resources/views/component/footersocial.blade.php (hunk 6 above)
Comment (apani) [inline resources/views/component/footersocial.blade.php]: lets go with footer/social.blade.php

File: resources/views/component/offer.blade.php (hunk 1 above)
Comment (apani) [inline resources/views/component/offer.blade.php]: will change it to flight

File: resources/views/component/travelletter.blade.php (hunk 2 above)
Comment (apani) [inline resources/views/component/travelletter.blade.php]: Reisikiri === travel letter :) - can change it to whatever is the best

File: resources/views/layouts/main.blade.php [hunk 9]
```diff
@@ -3,17 +3,19 @@
...
+
     </head>
     <body>{{ Analytics::render() }}
-        
-        <div class="utils-padding-bottom">
 
-            @yield('header1', view('component.header1'))
+        @include('component.header')
+
+{{--         @yield('masthead', view('component.masthead'))
```
Comment (kristjanjansen) [inline resources/views/layouts/main.blade.php]: Why comments?

File: resources/views/component/search.blade.php [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+    </div>
+
+    <div class="c-search__results">
+
+        <div class="c-search__results-content">
+
+            <ul class="c-search__results-list m-destinations">
+
+                <li class="c-search__results-list-item">
+                    Sihtkohad
```
Comment (apani) [inline resources/views/component/search.blade.php]: Dummy content from local dev files - will remove

File: resources/views/layouts/main.blade.php (hunk 9 above)
Comment (apani) [inline resources/views/layouts/main.blade.php]: will remove

File: resources/views/component/map.blade.php (hunk 7 above)
Comment (apani) [inline resources/views/component/map.blade.php]: How to porperly add standalone svg-s to the template?

File: resources/views/component/news.blade.php (hunk 8 above)
Comment (apani) [inline resources/views/component/news.blade.php]: whats the correct input for component.date.* ?

File: resources/views/component/map.blade.php (hunk 7 above)
Comment (kristjanjansen) [inline resources/views/component/map.blade.php]: Need to figure it out, #338

File: resources/views/component/news.blade.php (hunk 8 above)
Comment (kristjanjansen) [inline resources/views/component/news.blade.php]: #339

Comment (kristjanjansen) [inline resources/views/component/news.blade.php]: fixed

File: resources/views/component/travelletter.blade.php (hunk 2 above)
Comment (kristjanjansen) [inline resources/views/component/travelletter.blade.php]: Follow up on #341

**Pull request merged**